FEM/rf_num_new.cpp

```diff
@@ -698,6 +702,17 @@ ios::pos_type CNumerics::Read(ifstream* num_file)
 			     << "\n";
 			continue;
 		}
+		// automatic damping of Newton scheme
+		if (line_string.find("$AUTOMATIC_NEWTON_DAMPING") != string::npos) // NW
+		{
+			line.str(GetLineFromFile1(num_file));
+			line >> automatic_damping_tolerance; //if NR error decreases by less than this factor, the next step will be dampened
```

Comment:
Suggest to remove ``automatic_`` because that the damping factor is specified not calculated.